### PR TITLE
Resprites the base Coffeemaker, resprites coffeepots, adds bs pot overlay to both coffeemakers

### PR DESCRIPTION
**Pull request by ghost**
## About The Pull Request
image

The basic cartridge coffeemaker was pretty... crusty.
image

There was a [downstream resprite](https://github.com/Skyrat-SS13/Skyrat-tg/pull/20776) over a year ago that I suggested go upstream that was just never attempted. 
This isn't exactly the same as that though. I've modified the coffeemaker heavily from that and reshaped the pots a bit, and didn't include the cartridges.
**I did not change the one on the right.** That is unmodified. It is here for Comparison and to show the Bluespace Pot overlay.

Also two minor things:
- Lets you use the machine unpowered. It still can't brew without power, but now you can actually take the pot and cups off still. 
- The BS Coffeepot set a list to list(0) but the variable said to set it to null when not in use. It wasn't in use, so setting it to null hopefully prevents any weirdness with overlays trying to be made in the future.
## Why It's Good For The Game
I believe the sprites here are improvements. At least the coffeemaker and the pot lids are more legible.
## Changelog
:cl:
image: updated the cartridge coffeemaker + coffeepot sprites, and added bluespace pot overlays to coffeemakers
qol: the coffeemakers can now be interacted with while unpowered! No longer will your coffeepot be stuck inside the machine because it's unplugged! (Still need power to brew)
/:cl:

### CODE REVIEW

File: code/modules/food_and_drinks/machinery/coffeemaker.dm [hunk 1]
```diff
@@ -148,7 +148,12 @@
 /obj/machinery/coffeemaker/proc/overlay_checks()
 	. = list()
 	if(coffeepot)
-		. += "coffeemaker_pot"
+		if(istype(coffeepot, /obj/item/reagent_containers/cup/coffeepot/bluespace))
+			. += "coffeemaker_pot_bluespace"
+		else if(coffeepot.reagents.total_volume > 0)
+			. += "coffeemaker_pot_full"
+		else
+			. += "coffeemaker_pot_empty"
```
Comment (SyncIt21) [inline code/modules/food_and_drinks/machinery/coffeemaker.dm:156]:
```suggestion
		. += "coffeemaker_pot_[coffeepot.reagents.total_volume ? "full" : "empty"]"
```

**Review by Wallemations — APPROVED** [review]

File: code/modules/food_and_drinks/machinery/coffeemaker.dm (hunk 1 above)
Comment (OrionTheFox) [reply code/modules/food_and_drinks/machinery/coffeemaker.dm:156]: so wise are you...

**Comment by Wallemations** [comment]
Is this ready to merge?

**Comment by OrionTheFox** [comment]
> Is this ready to merge?

Yep!

**Pull request merged**